File: Codes/uysal_onur.php
<?php
for ($i = 0; $i < 3; $i++) {
    echo $i . "\n";
}

for ($i = 0; $i < 3; $i++) {
    echo $i . "\n";
    break;
}

for ($i = 0; $i < 3; $i++) {
    echo $i . "\n";
    if ($i == 1) {
        break;
    }
}

for ($i = 0; $i < 3; $i++) {
    continue;
    echo $i . "\n";
}

for ($i = 0; $i < 3; $i++) {
    if ($i == 1){
        continue;
    }
    echo $i . "\n";
}

for ($i = 0; $i < 3; $i++) {
    echo "outer loop iteration " . $i . "\n";
    for ($y = 0; $y < 3; $y++) {
        break;
        echo "inner loop iteration " . $y . "\n";
    }
    echo "outer loop after break" . "\n";
}

for ($i = 0; $i < 3; $i++) {
    echo "outer loop iteration " . $i . "\n";
    for ($y = 0; $y < 3; $y++) {
        continue;
        echo "inner loop iteration " . $y . "\n";
    }
    echo "outer loop after continue" . "\n";
}

?>

File: Codes/uysal_onur.rs
fn main() {
    for i in 0..3 {
        println!("{}", i);
    }
    
    for i in 0..3 {
        println!("{}", i);
        break;
    }
    
    for i in 0..3 {
        println!("{}", i);
        if i == 1 {
            break;
        }
    }
    
    for i in 0..3 {
        continue;
        println!("{}", i);
    }
    
    for i in 0..3 {
        if i == 1 {
            continue;
        }
        println!("{}", i);
    }
    
    for i in 0..3 {
        println!("outer loop iteration {}", i);
        for y in 0..3 {
            continue;
            println!("inner loop iteration {}", y);
        }
        println!("outer loop after continue");
    }
    
    for i in 0..3 {
        println!("outer loop iteration {}", i);
        for y in 0..3 {
            break;
            println!("inner loop iteration {}", y);
        }
        println!("outer loop after break");
    }
    println!("---------------------------");
    'outer: for i in 0..3 {
        println!("outer loop iteration {}", i);

        'inner: for y in 0..3 {
            println!("inner loop iteration {}", y);
            break 'outer;
        }
        println!("outer loop after break");
    }
    
    println!("---------------------------");
    'outer: for i in 0..3 {
        println!("outer loop iteration {}", i);

        'inner: for y in 0..3 {
            continue 'outer;
            println!("inner loop iteration {}", y);
        }
        println!("outer loop after break");
    }
    
}
